packages/bsp/mpv/mpv_mainline.conf

```diff
@@ -1,2 +1,3 @@
 # HW acceleration is not supported on this platform yet
 vo=x11
+sws-scaler=fast-bilinear
```

Comment:
This is the only change needed?

Reply:
Yes. This allows any device (where there is no HW) to play the video in full screen without heavy brakes.